.github/workflows/lychee.yml

```diff
@@ -1,36 +1,55 @@
-name: Lychee Link Checker
+name: Periodic Link Checker
 
-on: [ pull_request ]
+on:
+    schedule:
+        - cron: '0 0 * * *'  # Run daily at midnight UTC
+    workflow_dispatch:  # Allow manual triggering
 
 jobs:
     link-check:
         runs-on: ubuntu-latest
         steps:
-            -   uses: actions/checkout@v4
+            - uses: actions/checkout@v4
```

Comment:
please revert those whitespace changes too, the indent should be 4 spaces, not 4 and sometimes 2 at the end...

Reply:
idk how you format this, webstorm handles this correctly out of box, here is the full formatted file:

```yml
name: Periodic Link Checker

on:
    schedule:
        -   cron: '0 0 * * *'  # Run daily at midnight UTC
    workflow_dispatch:  # Allow manual triggering

jobs:
    link-check:
        runs-on: ubuntu-latest
        steps:
            -   uses: actions/checkout@v4

            -   name: Use Node.js 20
                uses: actions/setup-node@v4
                with:
                    node-version: 20
                    cache: 'npm'
                    cache-dependency-path: 'package-lock.json'
                    always-auth: 'true'
                    registry-url: 'https://npm.pkg.github.com/'
                    scope: '@apify-packages'

            -   name: Build docs
                run: |
                    npm ci --force
                    npm update @apify/openapi
                    npm run build
                env:
                    APIFY_SIGNING_TOKEN: ${{ secrets.APIFY_SIGNING_TOKEN }}
                    NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
                    SMARTLOOK_PROJECT_KEY: ${{ secrets.SMARTLOOK_PROJECT_KEY }}

            -   name: Run Lychee Link Checker
                id: lychee
                uses: lycheeverse/lychee-action@v1.10.0
                env:
                    GITHUB_TOKEN: ${{ secrets.APIFY_SERVICE_ACCOUNT_GITHUB_TOKEN }}
                with:
                    fail: false
                    args: >
                        --base https://docs.apify.com
                        --exclude-path 'build/versions.html'
                        --max-retries 6
                        --verbose
                        --no-progress
                        --accept '100..=103,200..=299,403..=403,429'
                        --output ./lychee/out.md
                        --format markdown
                        './build/**/*.html'
                continue-on-error: true

            -   name: Add job summary
                if: always()
                run: cat ./lychee/out.md >> $GITHUB_STEP_SUMMARY
```